ui/components/app/snaps/snap-ui-radio-group/snap-ui-radio-group.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { FunctionComponent, useEffect, useState } from 'react';
+import { useSnapInterfaceContext } from '../../../../contexts/snaps';
+import {
+  AlignItems,
+  Display,
+  FlexDirection,
+  TextVariant,
+} from '../../../../helpers/constants/design-system';
+import {
+  Box,
+  HelpText,
+  HelpTextSeverity,
+  Label,
+  Text,
+} from '../../../component-library';
+
+export type SnapUIRadioOption = { value: string; name: string };
+
+export type SnapUIRadioProps = {
+  name: string;
+  label?: string;
+  error?: string;
+  options: SnapUIRadioOption[];
+  form?: string;
+};
+
+export const SnapUIRadioGroup: FunctionComponent<SnapUIRadioProps> = ({
+  name,
+  label,
+  error,
+  form,
+  ...props
+}) => {
+  const { handleInputChange, getValue } = useSnapInterfaceContext();
+
+  const initialValue = getValue<string>(name, form);
```

Comment:
```suggestion
  const initialValue = getValue(name, form) as string;
```

I changed the typing for this in the base branch

Reply:
Updated.